lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -25,9 +25,12 @@ const URL_PREFIXES = ['http://', 'https://', 'data:'];
 class DetailsRenderer {
   /**
    * @param {DOM} dom
+   * @param {{fullPageScreenshot?: LH.Artifacts.FullPageScreenshot}} [options]
    */
-  constructor(dom) {
+  constructor(dom, options = {}) {
```

Comment:
`options` seems weird since it's not an options bag but just some extra shit.  `optional` seems OK until we know where this obj is going

Reply:
we have 11 instances of naming parameters like this `options`, none for `optional`. of those at least 5 have just one property.